### PR TITLE
[DOC] add examples/clarify CONTRIBUTORS.md

### LINKED ISSUE #1565: Suggestions for CONTRIBUTING.md
Just contributed for the first time. The CONTRIBUTING guide was very helpful, thanks! I have a few suggestions for the following steps:

- Step 3: add `git commit` and `git add` commands

- Step 4: 
  - include examples of what falls under each descriptive prefix
  - change link for https://help.github.com/en/articles/creating-a-pull-request to https://help.github.com/en/articles/creating-a-pull-request-from-a-fork

### PR DESCRIPTION
**Pull request by jdkent**
## Changes proposed in this pull request
fixes #1565 
- add example pull requests to each type (e.g. `STY`, `DOC`, `ENH`, etc)
- add link to add/commit/push files
- change pull request link to forked repositories (since that is the workflow that fmriprep promotes)
- add a cloning step (so contributors have explicit instructions on how to clone the repository)

## Documentation that should be reviewed

The CONTRIBUTORS.md file

@ariveradompenciel could you see if this updated [CONTRIBUTING.md](https://github.com/jdkent/fmriprep/blob/contrib_clarity/CONTRIBUTING.md) addresses your suggestions?

@effigies, @oesteban, @emdupre :  I do not know what `RTM` means (it's not Read The Manual), sorry I'm not keeping up with the lingo. But I would like to add an entry for that, which I can do as soon as I know what it means.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -229,11 +231,21 @@ You're awesome. :wave::smiley:
 [link_helpwanted]: https://github.com/poldracklab/fmriprep/labels/help%20wanted
 [link_feature]: https://github.com/poldracklab/fmriprep/labels/feature
 
-[link_pullrequest]: https://help.github.com/articles/creating-a-pull-request/
+[link_pullrequest]:https://help.github.com/en/articles/creating-a-pull-request-from-a-fork
```
Comment (jdkent) [inline CONTRIBUTING.md]: Forgot a space here and I should get rid of the `/en/` in the path name

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -229,11 +231,21 @@ You're awesome. :wave::smiley:
...
 [link_fork]: https://help.github.com/articles/fork-a-repo/
 [link_pushpullblog]: https://www.igvita.com/2011/12/19/dont-push-your-pull-requests/
 [link_branches]: https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/
+[link_add_commit_push]: https://help.github.com/en/articles/adding-a-file-to-a-repository-using-the-command-line
```
Comment (jdkent) [inline CONTRIBUTING.md]: Remove `/en/` here too.

**Comment by effigies** [comment]
Yeah, it would be ready-to-merge, but I'm not sure we ever actually use it. More often, we tag with WIP and just remove the tag when it's ready for review/merge.

**Comment by jdkent** [comment]
So it's kind of complimentary  with `WIP`, where a "currently being worked on" pull request would be labelled with `[WIP]`, and then when it's ready to review/merge, it would be marked with `[RTM]`. (except contributors typically do not need to be aware of this semantics or use these symbols). If I understood correctly, then I think I got it.

**Comment by ariveradompenciel** [comment]
Thank you! This definitely addresses my suggestions!

**Pull request merged**